### PR TITLE
fix: Editable Typography in Card title style problems

### LINKED ISSUE #27105: Editable Typography in Card title style problems
- [x] I have searched the [issues](https://github.com/ant-design/ant-design/issues) of this repository and believe that this is not a duplicate.

### Reproduction link
[Edit on CodeSandbox](https://codesandbox.io/s/editable-card-title-7hf5z?file=/src/App.js)

### Steps to reproduce
* Click on edit icon
* Enter long text

### What is expected?
Title should display with proper ellipsis

### What is actually happening?
Several style issues:
* Textarea is left shifted when edit is clicked
* On submitting long text 
    * Title is left shifted
    * Edit icon is not visible (only tootlip is visible)

| Environment | Info |
|---|---|
| antd | 4.7.0 |
| React | latest |
| System | MacOS |
| Browser | Chrome Version 86.0.4240.75 |

---
Also, tried using Typography ellipsis but it didn't solve the problem

### PR DESCRIPTION
**Pull request by hengkx**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md)]

### 🤔 This is a ...

- [ ] New feature
- [x] Bug fix
- [ ] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Other (about what?)

### 🔗 Related issue link
close #27105

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |  fix Editable Typography in Card title style problems.         |
| 🇨🇳 Chinese |  修复 Typography 在 Card title 中编辑样式问题。        |

### ☑️ Self Check before Merge

⚠️ Please check all items below before review. ⚠️

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

### CODE REVIEW

File: components/card/style/index.less [hunk 1]
```diff
@@ -57,6 +57,13 @@
       overflow: hidden;
       white-space: nowrap;
       text-overflow: ellipsis;
+
+      .@{ant-prefix}-typography,
+      .@{ant-prefix}-typography-edit-content {
+        left: 0;
```
Comment (hengkx) [inline components/card/style/index.less:63]: @afc163 不知道 当时 为什么 加了个left，没看出来为什么
https://github.com/ant-design/ant-design/blob/master/components/typography/style/index.less#L176

**Comment by hengkx** [comment]
codesand 这东西 好像挂了

File: components/card/style/index.less (hunk 1 above)
Comment (afc163) [reply components/card/style/index.less:63]: https://ant.design/components/typography-cn/#components-typography-demo-interactive

防止切换时文本跳动。

File: components/card/style/index.less [hunk 2]
```diff
@@ -57,6 +57,13 @@
       overflow: hidden;
       white-space: nowrap;
       text-overflow: ellipsis;
+
+      .@{ant-prefix}-typography,
```
Comment (afc163) [inline components/card/style/index.less:61]: 可以加个 `> .@{ant-prefix}-typography`。

**Pull request merged**